testapps/testapp/src/main/res/raw/msal_config.json

```diff
@@ -1,14 +1,14 @@
 {
   "client_id" : "4b0db8c2-9f26-4417-8bde-3f0e3656f8e0",
   "authorization_user_agent" : "DEFAULT",
-  "redirect_uri" : "msauth://com.microsoft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
+  "redirect_uri" : "msauth://com.msft.identity.client.sample.local/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
   "multiple_clouds_supported":true,
   "client_capabilities": "CP1",
   "authorities" : [
     {
       "type": "AAD",
       "audience": {
-        "type": "AzureADandPersonalMicrosoftAccount"
```

Comment:
This was not correct, this app didn't support personal account. I've added a new (app) config below that supports both organization and personal accounts.